src/server.ts

```diff
@@ -0,0 +1,24 @@
+// Import the 'express' module along with 'Request' and 'Response' types from express
+import express, { Request, Response } from "express";
+import cors from "cors";
+// Create an Express application
+const app = express();
+
+// Specify the port number for the server
+//const PORT = process.env.PORT || 3002;
+const PORT = __filename.endsWith(".ts") ? 3003 : 3001;
+
+app.use(cors({ origin: "http://localhost:3000" }));
```

Comment:
why do you keep static vars on the code? will we search alyaws  all codes for the static variable?! move it to contact file or to .env file

Reply:
created a file config.ts and moved the const PORT in there.